src/actors/upgrade/common/list_content_pkg/list_content_pkg.py

```diff
@@ -5,12 +5,11 @@
 import sys
 
 keys = {
+    'name': 'list_content_pkg',
     'in_ctx': 'context',
     'in_content': 'content',
     'out': 'pkg',
-    'err_out': 'error',
-    'err_ctx': 'context',
-    'err_value': 'value'
+    'check_out': 'check_output'
 }
```

Comment:
Just for future reference, this isn't related to this PR (i.e., no need to change anything here):

IMO keeping this data structure to reference keys makes it more difficult to follow the code (even though it makes easier to change the keys). For instance, I guess this would a more readable way of getting `content`:

```context = ','.join(x['value'] for x in inputs.get('context', []))```

Anyway, this is just personal preference.